deploy/helm/elastic-agent/values.yaml

```diff
@@ -0,0 +1,455 @@
+      # -- jobs state stream vars
+      # @section -- 2 - Kubernetes integration
+      vars: {}
+  cronjobs:
+    state:
+      # -- enable cronjobs state stream (kube-state-metrics) [ref](https://www.elastic.co/docs/current/integrations/kubernetes/kube-state-metrics#state_cronjob)
+      # @section -- 2 - Kubernetes integration
+      enabled: true
+      # -- cronjobs state stream vars
+      # @section -- 2 - Kubernetes integration
+      vars: {}
+  persistentvolumes:
+    state:
+      # -- enable persistentvolumes state stream (kube-state-metrics) [ref](https://www.elastic.co/docs/current/integrations/kubernetes/kube-state-metrics#state_persistentvolume)
+      # @section -- 2 - Kubernetes integration
+      enabled: true
+      # -- persistentvolumes state stream vars
+      # @section -- 2 - Kubernetes integration
+      vars: {}
+  persistentvolumeclaims:
+    state:
+      # -- enable persistentvolumeclaims state stream (kube-state-metrics) [ref](https://www.elastic.co/docs/current/integrations/kubernetes/kube-state-metrics#state_persistentvolumeclaim)
+      # @section -- 2 - Kubernetes integration
+      enabled: true
+      # -- persistentvolumeclaims state stream vars
+      # @section -- 2 - Kubernetes integration
+      vars: {}
+  resourcequotas:
+    state:
+      # -- enable resourcequotas state stream (kube-state-metrics) [ref](https://www.elastic.co/docs/current/integrations/kubernetes/kube-state-metrics#state_resourcequota)
+      # @section -- 2 - Kubernetes integration
+      enabled: true
+      # -- resourcequotas state stream vars
+      # @section -- 2 - Kubernetes integration
+      vars: {}
+  services:
+    state:
+      # -- enable services state stream (kube-state-metrics) [ref](https://www.elastic.co/docs/current/integrations/kubernetes/kube-state-metrics#state_service)
+      # @section -- 2 - Kubernetes integration
+      enabled: true
+      # -- services state stream vars
+      # @section -- 2 - Kubernetes integration
+      vars: {}
+  system:
+    metrics:
+      # -- enable system metric stream (kubelet) [ref](https://www.elastic.co/docs/current/integrations/kubernetes/kubelet#system)
+      # @section -- 2 - Kubernetes integration
+      enabled: true
+      # -- system metric stream vars
+      # @section -- 2 - Kubernetes integration
+      vars: {}
+
+# -- extra [user-defined integrations](https://www.elastic.co/guide/en/fleet/current/elastic-agent-input-configuration.html) to be added to the Elastic Agent
+# An example can be found [here](./examples/nginx-custom-integration/README.md)
+# @section -- 3 - User Extra Integrations
+extraIntegrations: {}
+
+agent:
+  # -- enable elastic-agent
+  # @section -- 3 - Elastic-Agent Configuration
+  enabled: true
+  # -- elastic-agent version
+  # @section -- 3 - Elastic-Agent Configuration
+  version: 8.15.0
+  # -- image configuration
+  # @section -- 3 - Elastic-Agent Configuration
+  image:
+    repository: docker.elastic.co/beats/elastic-agent-complete
```

Comment:
Why is `-complete` being used? `synthetics`? Seems like it would be better to not use that one unless they are wanting to do `synthetics`.

Reply:
maybe we want to support that?! if we don't this can be default to something else or maybe switch to wolfi-based images?!

Reply:
I would not default to `-complete` its much larger, just for synthetics. I would expect them to adjust the chart to use `-complete` if they want to run synthetics. Wolfi image could also be the default, no issues there from me.

Reply:
I think the open question here is how is this Helm chart getting released? Does it get updated out of sync of the Elastic Agent release or does it get released at the same time of the full stack release? If its stack release based then this should really sync with the version of the repository [here](https://github.com/elastic/elastic-agent/blob/main/version/version.go#L7) and when its the `main` repository it should really be `-SNAPSHOT` as that is the only released version for running code in actual `main`. When `8.16` is created, then it should not use `-SNAPSHOT`.

Reply:
Just a quick thinking about that; I would like having this chart "following" the agent release to always have the appropriate agent image in it, but also be able to bump helm chart version to fix any agent-unrelated issue, e.g. issues with templating?!

Reply:
Can we defer the release process question for now? It sounds like it could be its own major discussion, but whatever we decide is not going to require major changes to this Chart, which we're not publishing yet anyway. I think it might be better to merge this PR without answering this question, and do that in a separate issue. WDYT @blakerouse @pkoutsovasilis ?

Reply:
yep I agree with deferring the release process question for now. That said, I added a helm:updateAgentVersion mage target that I deem it would be useful in this discussion

Reply:
I agree we should defer. A follow up PR to change this is simple.